projects/angular/src/emphasis/alert/alert.ts

```diff
@@ -60,11 +61,11 @@ export class ClrAlert implements OnInit, OnDestroy {
   @Output('clrAlertClosedChange') _closedChanged: EventEmitter<boolean> = new EventEmitter<boolean>(false);
 
   @Input('clrAlertType')
-  set alertType(val: string) {
+  set alertType(val: AlertType) {
```

Comment:
Breaking change that people will complain about right away. See my comments below on the file that declares the `AlertType`